### PR TITLE
Final prep for Release 0.12

### PR DESCRIPTION
**Pull request by avijit-nervana**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Nervana Systems Inc.
+# Copyright 2019 Nervana Systems Inc.
```
Comment (sayantan-nervana) [inline CMakeLists.txt]:
```suggestion
# Copyright 2018-2019 Nervana Systems Inc.
```

File: build_ngtf.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # ==============================================================================
-#  Copyright 2018 Intel Corporation
+#  Copyright 2019 Intel Corporation
```
Comment (sayantan-nervana) [inline build_ngtf.py]:
```suggestion
#  Copyright 2018-2019 Intel Corporation
```

File: logging/CMakeLists.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Nervana Systems Inc.
+# Copyright 2019 Nervana Systems Inc.
```
Comment (sayantan-nervana) [inline logging/CMakeLists.txt]:
```suggestion
# Copyright 2018-2019 Nervana Systems Inc.
```

File: src/CMakeLists.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Nervana Systems Inc.
+# Copyright 2019 Nervana Systems Inc.
```
Comment (sayantan-nervana) [inline src/CMakeLists.txt]:
```suggestion
# Copyright 2018-2019 Nervana Systems Inc.
```

File: src/tf_deadness_analysis.h [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```
Comment (sayantan-nervana) [inline src/tf_deadness_analysis.h]:
```suggestion
/* Copyright 2018-2019 The TensorFlow Authors. All Rights Reserved.
```

File: test/python/CMakeLists.txt [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Nervana Systems Inc.
+# Copyright 2019 Nervana Systems Inc.
```
Comment (sayantan-nervana) [inline test/python/CMakeLists.txt]:
```suggestion
# Copyright 2018-2019 Nervana Systems Inc.
```

File: third-party/CMakeLists.txt [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Nervana Systems Inc.
+# Copyright 2019 Nervana Systems Inc.
```
Comment (sayantan-nervana) [inline third-party/CMakeLists.txt]:
```suggestion
# Copyright 2018-2019 Nervana Systems Inc.
```

File: README.md [hunk 8]
```diff
@@ -57,7 +57,7 @@ TensorFlow model scripts and running them the usual way:
 
         import ngraph_bridge
 
-Examples on how to use ngraph_bridge is here : https://github.com/NervanaSystems/ngraph-tf/tree/master/examples
+Detailed examples on how to use ngraph_bridge is located in the [examples] directory.
```
Comment (sayantan-nervana) [inline README.md]: `git checkout v0.12.0-rc6` needs to change to `git checkout v0.12.0`

Comment (avijit-nervana) [reply README.md]: Done.

File: README.md [hunk 9]
```diff
@@ -47,7 +47,7 @@ a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the
 This will produce something like this:
 
         TensorFlow version:  1.13.1
-        nGraph bridge version: b'0.12.0-rc6'
+        nGraph bridge version: b'0.12.0'
```
Comment (sayantan-nervana) [inline README.md]: Right now after `CXX11_ABI` prints we also get a new line: `nGraph bridge built with Grappler: True`
Should we update the README?

Comment (avijit-nervana) [reply README.md]: We will have t redo the README ... for now we will leave it as is

**Pull request merged**